knative-serving-1.16.yaml

```diff
@@ -0,0 +1,90 @@
+package:
+  name: knative-serving-1.16
+  version: 1.16.0
+  epoch: 0
+  description: Kubernetes-based, scale-to-zero, request-driven compute
```

Comment:
I think we should consider adding a provides here? `knative-serving=${{package.full-version}}` 💭